index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile first portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="top">
+      <div class="tim">
+        <h3 class="details clock">9:41</h3>
+      </div>
+      <div class="network">
+        <img src="./Img/Cellular Connection.svg" alt="">
+        <img src="./Img/Wifi.svg" alt="">
+        <img src="./Img/Battery.svg" alt="">
+      </div>
+    </div>
+    <div class="bottom">
+      <div class="logo">
+        <h2 class="details name">John Doe</h2>
+      </div>
+      <div class="nav-bar">
+        <img src="./Img/Union.svg" alt="">
+      </div>
+    </div>
+  </header>
+  <section class="head-line">
+    <div class="intro">
+      <h2 class="greet">Hey There. I’m Bran</h2>
+      <h3 class="job details">I’am a Software Developer</h3>
+      <p class="details we-do">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+    </div>
+    <div class="web-link">
+      <img src="./Img/git.svg" alt="">
+      <img src="./Img/Icon - Linkedin.svg" alt="">
+      <img src="./Img/Enabled.svg" alt="">
+      <img src="./Img/Icon - Twitter.svg" alt="">
+      <img src="./Img/Vector.svg" alt="">
+    </div>
```

Comment:
- [ ] I would advise you to organize this code better by using the ul tag and having the img tag in an li tag individually. Also, these are meant to link to your social apps, you can add the img to in an a tag so that you will take advantage of the href.